app/javascript/components/Payment/Payment.js

```diff
@@ -63,14 +64,22 @@ export class Payment extends Component {
 
   componentDidMount() {
     const urlInfo = window.champaign.personalization.urlParams;
-    const donor_status = window.champaign.personalization.member.donor_status;
+    const donor_status =
```

Comment:
If you're doing these checks to verify that the object has the given attribute, you can also look at the optional chaining for JavaScript for easier syntax - https://github.com/tc39/proposal-optional-chaining - so you could do `window.champaign.personalization.member?.donor_status` and `window.champaign.plugins.fundraiser?.default?.config.recurring_default`. No change required here, just something I was reminded of.

Reply:
I tried this syntax first. When my VSCode warned and thought we are using older ES so reverted back. Should check whats wrong with my VSCode, slow may be. Anyways this is done.